src/site/content/en/blog/centering-in-css/index.md

```diff
@@ -0,0 +1,347 @@
+created The Resilience Ringer, a series of stress tests for each centering
+strategy to balance within. I even pick a winner at the end, which is my most
+favorite and used centering technique of all time! 
+
+### The Resilience Ringer
+
+The Resilience Ringer is the implementation of my beliefs that a centering
+strategy should be resilient to international layouts, CMS editing, and dynamic
+content. These tenets helped shape the following resilience tests for the
+centering techniques to endure:
+
+1. **Squished:** centering should be able to handle changes to width
+2. **Squashed:** centering should be able to handle changes to height
+3. **Duplicate:** centering should be dynamic to number of items
+4. **Edit:** centering should be dynamic to length and language of content
+5. **Flow:** centering should be document direction and writing mode agnostic
+
+The winning solution will demonstrate its resilience by keeping its content in
+the center while it's squished, squashed, duplicated, edited, and swapped to
+various language modes and directions.
+
+#### Legend
+
+I've provided some visual color hinting to help you keep some meta information
+in context:
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="./legend.png" alt="">
+  <figcaption class="w-figcaption"></figcaption>
+</figure>
+
+1. A pink border indicates ownership of centering styles
+1. The grey box is the background on the container which seeks to have centered
+   items
+1. Each child has a white background color so you can see any effects the
+   centering technique has on child box sizes (if any)
+
+## The 5 Contestants
+5 centering techniques enter the Resilience Ringer, only one will receive the
+Resilience Crown 👸. 
+
+### Content Center
+
+<figure class="w-figure w-figure--fullbleed">
+  <video playsinline controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/atoms-sandbox.google.com.a.appspot.com/content-center-ringer-cycle.mp4">
+  </video>
+</figure>
+
+1. **Squish**: great!
+2. **Squash**: great!
+3. **Duplicate**: great!
+4. **Edit**: great!
+5. **Flow**: great!
+
+It's going to be hard to beat the conciseness of `display: grid` and the
+`place-content` shorthand. Since it centers and justifies children collectively,
+it's a solid centering technique for groups of elements meant to be read.
+
+```css
+.content-center {
+  display: grid;
+  place-content: center;
+  gap: 1ch;
+}
+```
+
+<div class="w-columns">
+{% Compare 'worse', 'Pros' %}
+- Content is centered even under constrained space and overflow  
+- Centering edits and maintenance are all in one spot  
+- Gap guarantees equal spacing amongst _n_ children  
+- Grid creates rows by default  
+
+{% endCompare %}
+
+{% Compare 'better', 'Cons' %}
+- The widest child (`max-content`) sets the width for all the rest. This will be
+  discussed more in [Gentle Flex](#gentle-flex).  
+
+{% endCompare %}
+</div>
```

Comment:
The `Pros` are red and the `Cons` are green? Please fix throughout the page

![image](https://user-images.githubusercontent.com/4713486/91222183-a7725f80-e6d3-11ea-8fc7-efc038bb43e5.png)